concepts/eventing/performance evaluation/nats-jetstream/different-cloud-providers/results.md

```diff
@@ -0,0 +1,113 @@
+# Performance Evalation of Eventing on different Cloud Providers
+
+## Table of Contents
+- Test Setup
+- Failure Types
+- Test Results
+- Screenshots of Crashes
+
+## Test Setup
+* Kyma CLI version: `2.6.2`
+* Kyma: 
+  * Production Profile
+  * JetStream with File Storage
+* K8s cluster:
+  * Kubernetes v1.24.3, v1.24.6
+  * Gardener cluster [Nodes: 2(min) to 6(max)]
+
+* Kyma deploy command:
+    ```
+    kyma deploy --source=main --profile production ...
+    ```
+* Testing tool: [K6](https://k6.io/) (deployed in the same Kyma cluster)
+
+## Failure Types:
+Here are the common crash failures occured during the testing:
+- F1  - Only one of the NATS Jetstream nodes storage is full
+- F2  - All JetStream node storage is full
+- F3  - NATS JetStream just doesn’t accept and dispatch events despite storage is not full. This is solely observed on Azure
+<br/><br/>
+
+## Tests Results
+In this section the test outcomes of the 2 day running test with the [K6](https://k6.io/). As you see the given K6 tool rps (request-per-second) is different at what is observed at EPP. Therefore, the actual value is EPP rps.
+As you see GCP and AWS performed similarly, also SSD and HDD difference was not much. HDD cannot be tested for AWS as it doesn't offer such storage type.
+### GCP
+CPU: 4, Memory: 15GB
+| cluster | K6 rps | EPP rps | JS Crashed | version | test duration | Failure type | NATS version |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| gcp1 | 500 | 320 | yes | 2.7.0-rc2 | 5 hours | full storage (F1) |  |
+| gcp2 | 250 | 232 | no | 2.7.0-rc2 |  |  |  |
+| gcp3 | 350 | 315 | yes | 2.7.0-rc2 | 5.5 hours | full storage (F2) |  |
+| gcp4 | 300 | 263 | yes | 2.7.0-rc2 | 13.5 hours | full storage (F2) |  |
+| gcp5 | 280 | 240 | yes | 2.7.0-rc2 | 31.5 hours |  |  |
+| gcp6 | 280 | 265 | yes | main |  | full storage (F2) | 2.9.0 |
+| gcphdd1 | 250 | 240 | no | 2.7.0-rc2 |  |  |  |
+| gcp-hdd | 300 | 278 | yes | main |  | full storage (F1) | 2.9.0 |
```

Comment:
For consistency:
- Let's mention the NATS version for all executions.
- Let's mention the test duration for all (if you still have this info).